tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py

```diff
@@ -862,7 +861,7 @@ def eltwise_unary_rdiv_trunc(
     **kwargs,
 ):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttnn.rdiv(t0, value, round_mode="trunc", memory_config=output_mem_config)
+    t1 = ttnn.rdiv(t0, round_mode="trunc", memory_config=output_mem_config)
 
```

Comment:
Why is rdiv being updated here?   What does this have to do with bitwise_not?

Reply:
Oops thanks! I mistook that. Now I just patched on the intended spot.